Controller/CategoriesFrontEndController.php

```diff
@@ -148,4 +147,25 @@ public static function test_input($data)
         $data = htmlspecialchars($data);
         return $data;
     }
+    
+    public static function renderError(string $templatePath, array $data)
+    {
+        if(file_exists($templatePath)){
+            //Extracts vars to current view scope
+            extract($data);
+
```

Comment:
I would rather extract this logic into a class that would be for rendering templates